person_class.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'nameable_class'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(book)
+    rental = Rental.new(data, book, self)
+    book.add_rental(rental)
+  end
```

Comment:
- [Optional] Dear @zbahati, while creating a rental, consider a setuation where the `date` is not the current date. Imagine the book is borrowed a day before, but its bing registered today. I would suggest accepting the `date` attribute as a parameter and creating the rental with it.